README.md

```diff
@@ -18,14 +18,16 @@ There are three modes cargo-sort can be used in:
  * **-g or --grouped**
     - When sorting keep table key value spacing. If you have dependency groups they will stick but be sorted within the grouping.
     The `key_value_newlines` config option needs to be `true` for this to have any effect.
-* **-p or --print**
+ * **-p or --print**
     - Write the sorted toml file to stdout.
-* **-w or --workspace**
+ * **-w or --workspace**
     - Checks every crate in the workspace based on flags. Only one root may be given.
+ * **-o or --order**
+    - Specify an ordering of tables. All nested tables will be sorted and appear after the specified table. Any unspecified table will be after specified.
```

Comment:
The introduction says that cargo-sort only sorts dependencies, this says otherwise. Am I understanding correctly that without this option, only entries of `dependencies`, `dev-dependencies`, `build-dependencies` are sorted and with say `-o package,features,dependencies`, the entries of `package` are sorted too?

Reply:
Only the entries of `dependencies`, `dev-dependencies`, `build-dependencies` and `workspace.member / exclude` will ever be sorted. What I'm trying to get across is that the `-o=package,features,dependencies` will only order the top-level tables and if you have

```toml
[dependencies]

[features]

[dependencies.foo]

# You get

[features]
ordered = false

[dependencies]
ordered = true

[dependencies.foo]
ordered = true
```

The word nested is probably not appropriate there? "Specify an ordering of top-level tables. Tables will appear in the order specified with sub-tables appearing after its parent. Any unspecified table will be after specified."

Reply:
Yes, the text you just suggested makes everything clear 👍🏼